k8s/kops/testground-daemon/sync-service.yml

```diff
@@ -0,0 +1,36 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: testground-sync-service
+spec:
+  selector:
+    matchLabels:
+      name: testground-sync-service
+  template:
+    metadata:
+      labels:
+        name: testground-sync-service
+    spec:
+      terminationGracePeriodSeconds: 10
+      hostPID: true
+      hostNetwork: true
+      dnsPolicy: ClusterFirstWithHostNet
+      containers:
+        - name: testground-sync-service
+          image: iptestground/sync-service:latest
+          command: ["/service"]
+          ports:
+            - name: syncservicehttp
+              containerPort: 5050
+          env:
+            - name: DEBUG
+              value: "true"
+          resources:
+            limits:
+              cpu: 240m
```

Comment:
Explained [here](https://github.com/testground/testground/issues/1453).

Reply:
Let's document what limits this setting enables and, of course, link to the issue. Also, the issue mentions `120m` while here we're using twice as much. Do we have data backing up the change?

Reply:
Added a comment to the yaml. This comment should be in a different file, my mistake - `/eks/yaml/tg-sync-service-deployment.yml`

So the 120m refers to the `requests` values, or how much the pod initially requests from the server. The `limits` values refer to the maximum amount of resources the cluster will save for this pod in case it needs to scale even further.

We do not have exact data backing up the change, it was a trial and error testing to see which values would allow us to run bigger storm tests without impacting the cluster.

Reply:
Can we include information on how the limits translate to maximum supported load in the comment?

Reply:
I don't think we have this measured, because we are now hitting the maximum hardware limits of the worker nodes. So we know for sure that this is enough to support that.

Reply:
Do we have hardware limits documented then? In that case, a note that there is no point increasing those further might make sense.

Reply:
Yeah, we will add a note clarifying that.